src/applications/vaos/tests/list/past-appointments.unit.spec.jsx

```diff
@@ -36,6 +42,53 @@ describe('VAOS integration: past appointments', () => {
     expect(queryByText(/Past 3 months/i)).to.exist;
   });
 
+  it('should update range on dropdown change', async () => {
+    const olderAppt = getVAAppointmentMock();
+    olderAppt.attributes = {
+      ...olderAppt.attributes,
+      startDate: moment()
+        .subtract(3, 'months')
+        .format(),
+      clinicFriendlyName: 'Some clinic',
+      facilityId: '983',
+      sta6aid: '983GC',
+    };
+
+    mockPastAppointmentInfo({ va: [] });
+
+    const { findByText, baseElement, queryByText } = renderInReduxProvider(
+      <PastAppointmentsList />,
+      {
+        initialState,
+        reducers,
+      },
+    );
+
+    expect(queryByText(/Loading your appointments/i)).to.exist;
+    expect(await findByText(/You don’t have any appointments/i)).to.exist;
```

Comment:
I don't think you need the `expect` around this. If you `await` `findByText`, if it can't find the text it'll throw an error, which will blow up the test before `expect` does anything.